src/server/database/initializer.js

```diff
@@ -15,6 +15,7 @@ module.exports = function(db) {
 				timestamp_refresh_last INTEGER,
 				time_refresh_expire INTEGER,
 				scope TEXT,
+				is_banned INTEGER,
```

Comment:
This flag doesn't have a default value set in `_insertNewUserEmbed` and `_insertNewUserDiscord`.

Reply:
Not really necessary since it only checks for `is_banned != 1` (in which case both `0` and `null` would return true)
Any arguments on why it should have a default value?

Reply:
Because the value is assumed to be an integer (`0` in this case). With the current check this isn't a problem, but it could be in the future if our code changes. If we checked `is_banned == 0` instead, we get either false or true depending on whether `is_banned` is `null` or `0`, and both are currently possible.
If we initialise the value, we can safely assume it's the type we want it to be.

Reply:
Fair enough!